### ISSUE TITLE
Get OpenCV working on the Raspberry pi

### ISSUE THREAD

**Issue by Kheiden** [Kheiden/RPi-tankbot#11]
pip install opencv-python completed successfully, but when attempting "import opencv" on the CLI, I received the following error:

ImportError: libcblas.so.3gf: cannot open shared object file: No such file or directory

After some research, it looks like I'll need to compile opencv from source:
https://www.pyimagesearch.com/2015/07/27/installing-opencv-3-0-for-both-python-2-7-and-python-3-on-your-raspberry-pi-2/
https://raspberrypi.stackexchange.com/questions/69169/how-to-install-opencv-on-raspberry-pi-3-in-raspbian-jessie

**Comment by Kheiden** [comment]
Building from source failed.

I will try the following:
https://github.com/opencv/issues/8878

**Comment by Kheiden** [comment]
Yesterday I attempted a clean build from source using the above linked guide. My computer shut down thus the ssh connection was terminated.

I need to look into the RPi logs to find out if it built successfully or not.

**Comment by Kheiden** [comment]
I'm pretty sure that the build failed on the Raspberry pi due to running out of space on the microSD card.  I'll be upgrading from an 8 GB to a 32 GB and attempting to rebuild the library.

**Comment by Kheiden** [comment]
After copying the data from the 8gb, perform the following:
RPi resize flash partitions

**Comment by Kheiden** [comment]
Success! After nearly two weeks, I was finally able to build opencv on my raspberry pi.

I had to reinstall and set up the OS on the raspberry pi with a 32gb because I was unable to extend the partition from 8gb to 32gb.

I also ran into issues compiling opencv due to running out of RAM, so I had to increase the swap space.

I used the below tutorial to assist in the building:
https://www.pyimagesearch.com/2017/09/04/raspbian-stretch-install-opencv-3-python-on-your-raspberry-pi/
